aspnetcore/fundamentals/websockets.md

```diff
@@ -15,35 +16,34 @@ uid: fundamentals/websockets
 
 By [Tom Dykstra](https://github.com/tdykstra) and [Andrew Stanton-Nurse](https://github.com/anurse)
 
-This article explains how to get started with WebSockets in ASP.NET Core. [WebSocket](https://wikipedia.org/wiki/WebSocket) is a protocol that enables two-way persistent communication channels over TCP connections. It's used for applications such as chat, stock tickers, games, anywhere you want real-time functionality in a web application.
+This article explains how to get started with WebSockets in ASP.NET Core. [WebSocket](https://wikipedia.org/wiki/WebSocket) ([RFC 6455](https://tools.ietf.org/html/rfc6455)) is a protocol that enables two-way persistent communication channels over TCP connections. It's used in apps that benefit from fast, real-time communication, such as chat, dashboard, and game apps.
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/websockets/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample)). See the [Next Steps](#next-steps) section for more information.
 
-
 ## Prerequisites
 
-* ASP.NET Core 1.1 (doesn't run on 1.0)
-* Any OS that ASP.NET Core runs on:
+* ASP.NET Core 1.1 or later
+* Any OS that supports ASP.NET Core:
   
-  * Windows 7 / Windows Server 2008 and later
+  * Windows 7 / Windows Server 2008 or later
   * Linux
   * macOS
 
-* **Exception**: If your app runs on Windows with IIS, or with WebListener, you must use:
+* **Exception**: If your app runs on Windows with IIS, or with HTTP.sys, you must use:
 
   * Windows 8 / Windows Server 2012 or later
   * IIS 8 / IIS 8 Express
-  * WebSocket must be enabled in IIS
+  * WebSockets must be enabled in IIS (See the [IIS/IIS Express support](#iisiis-express-support) section.)
```

Comment:
Just to add additional confusion, this last 2 points don't apply to HTTP.sys. For HTTP.sys, you just need to be running the right OS. IIS is not required and neither is the WebSockets module :)